CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.1.19
+ - Fix ability to set SSL protocols
```

Comment:
Its minor, but when you look a few month in the future its handy to be able to click on the issue.